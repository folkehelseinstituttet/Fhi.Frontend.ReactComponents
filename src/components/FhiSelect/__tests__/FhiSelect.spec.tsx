import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Select from '../FhiSelect';

describe('FhiSelect', () => {
  test('snapshot test', () => {
    const { container } = render(
      <Select
        id="selectId"
        label="Select"
        onChange={() => {}}
        value={5}
        options={[
          { id: '1', text: '1', value: 1 },
          { id: '2', text: '2', value: 2 },
          { id: '3', text: '3', value: 3 },
          { id: '4', text: '4', value: 4 },
          { id: '5', text: '5', value: 5 },
        ]}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
