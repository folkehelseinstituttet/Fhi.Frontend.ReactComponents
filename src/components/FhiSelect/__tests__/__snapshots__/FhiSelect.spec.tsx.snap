// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FhiSelect snapshot test 1`] = `
<div
  className=""
>
  <label
    className="form-label"
    htmlFor="selectId"
  >
    Select
  </label>
  <select
    className="form-select"
    id="selectId"
    onChange={[Function]}
    value={5}
  >
    <option
      value={1}
    >
      1
    </option>
    <option
      value={2}
    >
      2
    </option>
    <option
      value={3}
    >
      3
    </option>
    <option
      value={4}
    >
      4
    </option>
    <option
      value={5}
    >
      5
    </option>
  </select>
</div>
`;
